open-apparel-registry/open-apparel-registry
# Add delete and merge facilities UI

## Overview

Add UI enabling administrators to merge and delete facilities through the dashboard.

Connects #598 
Connects #599 

## Demo

![delete](https://user-images.githubusercontent.com/4165523/59716652-6b489280-91e4-11e9-87a3-b43eb37e602d.gif)

![merge](https://user-images.githubusercontent.com/4165523/59716653-6b489280-91e4-11e9-8407-db749ea094a5.gif)

## Notes

This currently uses the facility details endpoint to fetch facility data, then shows a list of contributors as is done in the details sidebar. If this suffices for showing "matches" then we can stick with that endpoint; otherwise we'll possibly need to update or supplement the existing details endpoint.

The delete and merge Redux actions currently just do `Promise.resolve` and then simulate deleting the data. When the API endpoints are available for merge and delete, we can wire this up.

## Testing Instructions

- serve this branch, then sign in as a super user and visit the dashboard
- click on the delete a facility link and verify that it works as expected
- go back to the dashboard and try out the merge two facilities link and verify that that also works as expected.

## Checklist
